README.md

```diff
@@ -65,7 +65,12 @@ The following configuration options have defaults, so the configuration wizard d
 * the pull branch strategy
   * how to sync the main branch / perennial branches with their upstream
   * default: `rebase`
-  * possible values: `merge`, `rebase`
+  * possible values: `rebase`, `merge`
```

Comment:
I understand default first but since we already list default in a separate row, I'd prefer keeping alpha order.